src/components/SideNav.tsx

```diff
@@ -133,15 +135,15 @@ const NavLink: React.FC<IPropsNavLink> = ({ item, path }) => {
   )
 }
 
-export interface IProps {
+export interface SideNavProps {
   path: string
 }
 
 // TODO set tree state based on if current path is a child
 // of the given parent. Currently all `path` items default to open
 // and they only collapse when clicked on.
 // e.g. solution: https://github.com/hasura/gatsby-gitbook-starter/blob/5c165af40e48fc55eb06b45b95c84eb64b17ed32/src/components/sidebar/tree.js
-const SideNav: React.FC<IProps> = ({ path }) => {
+const SideNav: React.FC<SideNavProps> = ({ path }) => {
```

Comment:
Since we are deprecating `React.FC`, now we should do
```suggestion
const SideNav = ({ path }: SideNavProps) => {
```